aws_cloudtrail_rules/aws_key_compromised.py

```diff
@@ -0,0 +1,14 @@
+def rule(event):
+    # Capture PutUserPolicy
+    request = event.get('requestParameters')
+    return event.get('eventName').startswith(
```

Comment:
I'm not seeing any other events that could match `PutRolePolicy` ([ref](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_identityandaccessmanagement.html)), should we just do an equality check?